src/index.html

```diff
@@ -5,8 +5,256 @@
+        <p class="product__code">Product code: 195434</p>
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star" data-qa="card"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product__rating-reviews">Reviews: 5</p>
+        </div>
+        <div class="product__price">
+          <p class="product__price-name">Price:</p>
+          <p class="product__price-sum">$2,199</p>
+        </div>
+        <a class="product__button" data-qa="card-hover">Buy</a>
+      </div>
+
+      <div class="product">
+        <img class="product__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+        >
+        <p class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="product__code">Product code: 195434</p>
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star" data-qa="card"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product__rating-reviews">Reviews: 5</p>
+        </div>
+        <div class="product__price">
+          <p class="product__price-name">Price:</p>
+          <p class="product__price-sum">$2,199</p>
+        </div>
+        <a class="product__button" data-qa="card-hover">Buy</a>
+      </div>
+
+      <div class="product">
+        <img class="product__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+        >
+        <p class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="product__code">Product code: 195434</p>
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star" data-qa="card"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="product__rating-reviews">Reviews: 5</p>
+        </div>
+        <div class="product__price">
+          <p class="product__price-name">Price:</p>
+          <p class="product__price-sum">$2,199</p>
+        </div>
+        <a class="product__button" data-qa="card-hover">Buy</a>
+      </div>
+
+      <div class="product">
+        <img class="product__img"
+        src="./images/imac.jpeg"
+        alt="imac"
+        >
+        <p class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="product__code">Product code: 195434</p>
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star" data-qa="card"></div>
```

Comment:
Why does star has data-qa card attribute??
Remove it from all stars and put it in a proper place